app/assets/javascripts/cvss.js

```diff
@@ -0,0 +1,13 @@
+(function() {
```

Comment:
this should do ... ideally use a class to make this `.datepicker` or check if someone already did that
```js
jQuery(function() {
   $("#datepicker1, #datepicker2").datepicker();
})
```